src/covid/home/summary-tiles/covid-summary-tiles.component.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { getReportingCohort } from '../../../api/api';
+import OHRIProgrammeSummaryTiles from '../../../components/tile/ohri-programme-summary-tiles.component';
+import { clientsEnrolledToCare } from '../../../constants';
+
+function CTSummaryTiles({ launchWorkSpace }) {
+  const { t } = useTranslation();
+  const [activeClientsCount, setActiveClientsCount] = useState(100);
+
+  useEffect(() => {
+    // TODO: cohort needs to be created.
+    // getReportingCohort(clientsEnrolledToCare).then(data => {
+    //   setActiveClientsCount(data.members.length);
+    // });
+  }, []);
```

Comment:
Minor one, if the commented out code isn't needed for future use I would just delete the whole `useEffect`